PrivateCollections/Ethiopia/MamherDegguAddet/MDA003.xml

```diff
@@ -30,42 +31,260 @@
+                                      <item xml:id="q3" n="2">
+                                          <num value="2">፪</num>
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="13r" to="20v"></locus>
+                                      </item>
+                                      <item xml:id="q4" n="3">
+                                          <num value="3">፫</num>
+                                          <dim unit="leaf">6</dim>
+                                          <locus from="21r" to="26v"></locus>
+                                      </item>
+                                      <item xml:id="q5" n="4">
+                                          <num value="4">፬</num>
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="27r" to="34v"></locus>
+                                      </item>
+                                      <item xml:id="q6" n="5">
+                                          <num value="5">፭</num>
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="35r" to="42v"></locus>
+                                      </item>
+                                      <item xml:id="q7" n="6">
+                                          <num value="6">፮</num>
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="43r" to="50v"></locus>
+                                      </item>
+                                      <item xml:id="q8" n="7">
+                                          <num value="7">፯</num>
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="51r" to="58v"></locus>
+                                      </item>
+                                      <item xml:id="q9" n="8">
+                                          <num value="8">፰</num>
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="59r" to="66v"></locus>
+                                      </item>
+                                      <item xml:id="q10" n="9">
+                                          <num value="9">፱</num>
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="67r" to="74v"></locus>
+                                      </item>
+                                      <item xml:id="q11" n="10">
+                                          <num value="10">፲</num>
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="75r" to="83v"></locus>
+                                          <note>The folio 77 was photographed twice as 77 and 78.</note>
```

Comment:
I think the foliation should be consistent. Folio 77 should be numbered 77 only, even if photographed twice. q11 runs from f. 75 to f. 82 then. The remaining quires need to be adjusted accordingly.

Reply:
Yes, you are right. It would be better if I do as you suggested but I have to work with the existing set of images.

Reply:
If the existing set of images cannot be changed, you need to make a note about this under "Foliation". This is a foliation mistake. "Fol. 77 was photographed twice and was assigned the number twice (fol. 77 and fol. 78)". Fol. 78 will be then not considered.

Reply:
Of course it can be changed if requested, I did not receive such a request. But then the rest would have to be renumbered. The advantage of digital numbering is the flexibility.